internal/constants.go

```diff
@@ -19,7 +19,7 @@ const (
 	BootTimeoutDefault   = 30000
 	ClientMonitorDefault = 15000
 	ConfigFileName       = "configuration.toml"
-	ConfigV2Stem         = "config/V2/"
+	ConfigRegistryStem   = "edgex/core/1.0"
```

Comment:
Missing slash at the end. Result is as follows in Consul.

<img width="375" alt="screen shot 2019-01-08 at 12 15 47 pm" src="https://user-images.githubusercontent.com/1342980/50850350-40a60400-133f-11e9-9db9-19d17c29a69c.png">